containers/containers.cabal

```diff
@@ -35,7 +35,7 @@ Library
     default-language: Haskell2010
     build-depends: base >= 4.6 && < 5, array >= 0.4.0.0, deepseq >= 1.2 && < 1.5
     hs-source-dirs: src
-    ghc-options: -O2 -Wall
+    ghc-options: -O2 -Wall -fwarn-incomplete-uni-patterns -fwarn-incomplete-record-updates
```

Comment:
Why `-fwarn...` instead of `-W...`?

Reply:
Because `-W` is post 8.0 only. I didn't actually check whether containers is supposed to support 7.x still, I just took the conservative option.